delphi-frontend/src/main/java/au/com/integradev/delphi/antlr/ast/node/AttributeNodeImpl.java

```diff
@@ -35,9 +37,14 @@ public AttributeNodeImpl(int tokenType) {
     super(tokenType);
   }
 
+  @Override
+  public boolean isAssembly() {
+    return getChild(0).getTokenType() == DelphiTokenType.ASSEMBLY;
+  }
+
   @Override
   public NameReferenceNode getNameReference() {
-    return (NameReferenceNode) getChild(0);
+    return (NameReferenceNode) getChild(isAssembly() ? 2 : 0);
```

Comment:
I think this should be `isAssembly() ? 1 : 0`, right? Seeing as this is the rewritten tree:
```
^(TkAttribute<AttributeNodeImpl> ASSEMBLY? nameReference argumentList? (':' nameReference argumentList?)*)
```

Reply:
Ack, an oversight that happened when I pruned the colons from the rewritten tree later on.
Thanks, I'll fix these.